mst/backup-types.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Types of Backups
```

Comment:
```suggestion
title: Types of backups in Managed Service for TimescaleDB
```
Is should be MST throughout, right? The entire KB is MST if I remember correctly, so the references to Cloud should be out of date (from before the rename)